homeassistant/components/homee/cover.py

```diff
@@ -112,7 +115,13 @@ def __init__(self, node: HomeeNode, entry: HomeeConfigEntry) -> None:
         )
         self._attr_device_class = get_device_class(node)
 
-        self._attr_unique_id = f"{self._attr_unique_id}-{self._open_close_attribute.id}"
+    @property
+    def unique_id(self) -> str:
```

Comment:
We can still do this in a shorthand right?

Reply:
It looked a little hard to read in the first moment to me, but after giving it another try, it works out.
```suggestion
        )

```

Reply:
hmm, the suggestion did not work as intended - I'll update the pr in a moment.